File: charArrayAndString/transNumberSystem.cpp
#include<iostream>
#include<cstdlib>
using namespace std;
int main(){
	int i = 65;
	char str[50];
	cout << "Binary = " << itoa(i, str, 2) << endl;
	cout << "Octal = " << itoa(i, str, 8) << endl;
	cout << "Hexadecimal = " << itoa(i, str, 16) << endl;
	return 0;
}

File: charArrayAndString/transNumberSystem.java
package springIntro;

import java.util.Scanner;

public class transNumberSystem {

	public static void main(String[] args) {
		int number;
		System.out.println("Enter a number");
		Scanner in  = new Scanner(System.in);
		number = in.nextInt();
		System.out.println("Binary equivalent of a number = " + Integer.toBinaryString(number));
		System.out.println("Octal equivalent of a number = " + Integer.toOctalString(number));
		System.out.println("Hexadecimal equivalent of a number = " + Integer.toHexString(number));
	}

}
